carta/session.py

```diff
@@ -418,6 +418,57 @@ def clear_raster_scaling_reference(self):
         """Clear the raster scaling reference."""
         self.call_action("clearRasterScalingReference")
 
+    # VIEWER MODES
+    @validate(String())
+    def set_viewer_mode(self, panel="single"):
```

Comment:
just a note for the future, there will be a new mode as "channel" for channel map view.

Reply:
In e62f8c4:
1. `mode` is used instead of `panel` for the argument.
2. Remove the default setting of `mode`.
3. Adopt `OneOf(["single", "multiple"])` instead of `String()` for the validation of `mode`.